lib/recart-nacelle-plugin.js

```diff
@@ -37,6 +37,19 @@ const initRecartPlugin = async (ctx) => {
         break
     }
   })
+
+  const cartId = ctx.query.cart_id
+  const utmSource = ctx.query.utm_source
```

Comment:
nit: ctx?.query

`query` is always present?

Reply:
it's provided by nuxt, it should be always present